config/autoscaler-service.yaml

```diff
@@ -30,7 +30,7 @@ spec:
     port: 9090
```

Comment:
should the `metrics` port above change to `http-metrics` as well?

Reply:
oh yep, missed it. updated, also other service yaml to comply with naming.